README.md

```diff
@@ -4,3 +4,37 @@ A version of [font-roboto](https://github.com/PolymerElements/font-roboto) using
 locally available font files
 
 The font files are subject to the Apache License found here: https://github.com/google/fonts/blob/master/apache/roboto/LICENSE.txt
+
+## Usage in a Polymer 2.x Application
```

Comment:
I think we can change this to just "How to use", since it applies to both 1.x and 2.x apps